### PR TITLE
update Snowflake destination docs with more info

### PR DESCRIPTION
**Pull request by subodh1810**
Andy suggested these improvements to our Snowflake documentation

### CODE REVIEW

File: docs/integrations/destinations/snowflake.md [hunk 1]
```diff
@@ -27,9 +27,15 @@ Note that Airbyte will create **permanent** tables. If you prefer to create tran
 
 ## Getting started
 
+### Requirements
+
+1. Active Snowflake warehouse
+2. A staging S3 or GCS bucket with credentials \(for the Cloud Storage Staging strategy\).
```
Comment (sherifnada) [inline docs/integrations/destinations/snowflake.md:33]: I think we should remove this since everyone should use internal staging

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/destinations/snowflake.md [hunk 2]
```diff
@@ -106,14 +112,14 @@ commit;
...
 
-* **Host**
-* **Role**
-* **Warehouse**
-* **Database**
-* **Schema**
-* **Username**
-* **Password**
-* **JDBC URL Params** (Optional)
+* **Host** : The host domain of the snowflake instance (must include the account, region, cloud environment, and end with snowflakecomputing.com). Example - `accountname.us-east-2.aws.snowflakecomputing.com`
```
Comment (misteryeo) [inline docs/integrations/destinations/snowflake.md:115]: For this section of descriptions, are external docs from Snowflake that can/should be referenced?

For example, with "Role", this seems like it's appropriate: https://docs.snowflake.com/en/user-guide/security-access-control-overview.html#roles.

File: docs/integrations/destinations/snowflake.md [hunk 3]
```diff
@@ -147,17 +153,17 @@ When an identifier is double-quoted, it is stored and resolved exactly as entere
...
-## Cloud Storage Staging
+## Loading Method
 
-By default, Airbyte uses batches of `INSERT` commands to add data to a temporary table before copying it over to the final table in Snowflake. This is too slow for larger/multi-GB replications. For those larger replications we recommend configuring using cloud storage to allow batch writes and loading.
+By default, Airbyte uses `INTERNAL STAGING`  
 
 ### Internal Staging
 
-Internal named stages are storage location objects within a Snowflake database/schema. Because they are database objects, the same security permissions apply as with any other database objects. No need to provide additional properties for internal staging
+Internal named stages are storage location objects within a Snowflake database/schema. Because they are database objects, the same security permissions apply as with any other database objects. No need to provide additional properties for internal staging. This is also the recommended way of using the connector.
```
Comment (misteryeo) [inline docs/integrations/destinations/snowflake.md:162]: Is there a reason why we recommend this and why this is default?

File: docs/integrations/destinations/snowflake.md [hunk 4]
```diff
@@ -27,9 +27,15 @@ Note that Airbyte will create **permanent** tables. If you prefer to create tran
...
 
+### Requirements
+
+1. Active Snowflake warehouse
+2. A staging S3 or GCS bucket with credentials \(for the Cloud Storage Staging strategy\).
+
 We recommend creating an Airbyte-specific warehouse, database, schema, user, and role for writing data into Snowflake so it is possible to track costs specifically related to Airbyte \(including the cost of running this warehouse\) and control permissions at a granular level. Since the Airbyte user creates, drops, and alters tables, `OWNERSHIP` permissions are required in Snowflake. If you are not following the recommended script below, please limit the `OWNERSHIP` permissions to only the necessary database and schema for the Airbyte user.
 
 We provide the following script to create these resources. Before running, you must change the password to something secure. You may change the names of the other resources if you desire.
+Login into your Snowflake warehouse, copy and paste the following script in a new [worksheet](https://docs.snowflake.com/en/user-guide/ui-worksheet.html). Select the `All Queries` checkbox and then press the `Run` button.
```
Comment (misteryeo) [inline docs/integrations/destinations/snowflake.md:38]: Yes! Love that we're hyperlinking out to Snowflake docs here!

**Review by misteryeo — CHANGES_REQUESTED** [review]
Looks great! Added a few comments - happy to continue discussing too. Also looped in Amruta as an FYI so she can see how things are changing with docs.

File: docs/integrations/destinations/snowflake.md (hunk 1 above)
Comment (subodh1810) [reply docs/integrations/destinations/snowflake.md]: Yes thats correct @misteryeo

**Review by misteryeo — APPROVED** [review]
LGTM!

**Pull request merged**